.github/workflows/iroha2-dev-pr.yml

```diff
@@ -137,3 +137,70 @@ jobs:
           file: Dockerfile
           # This context specification is required
           context: .
+
+  client-cli-test:
+    runs-on: ubuntu-latest
+    container:
+      image: hyperledger/iroha2-ci:nightly-2023-06-25
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Archlinux environment
+        run: |
+          sudo pacman -Syy
+          sudo pacman -S archlinux-keyring --noconfirm --disable-download-timeout
+          sudo rm -rf /etc/pacman.d/gnupg/* && sudo pacman-key --init && sudo pacman-key --populate archlinux
+          sudo pacman -Syu --noconfirm --disable-download-timeout
+
+      - name: Set up Rust toolchain and required dependencies
+        run: |
+          sudo pacman -S rustup mold musl rust-musl wget --noconfirm --disable-download-timeout
+          rustup toolchain install nightly-2023-06-25
+          rustup default nightly-2023-06-25
+          rustup target add x86_64-unknown-linux-musl wasm32-unknown-unknown
+          rustup component add rust-src
+          wget -c http://musl.cc/x86_64-linux-musl-native.tgz -O - | tar -xz
+          sudo ln -s /x86_64-linux-musl-native/bin/x86_64-linux-musl-g++ /x86_64-linux-musl-native/bin/musl-g++
+          sudo ln -s /x86_64-linux-musl-native/bin/x86_64-linux-musl-gcc-ar /x86_64-linux-musl-native/bin/musl-ar
+          sudo ln -s /x86_64-linux-musl-native/bin/x86_64-linux-musl-gcc-ranlib /x86_64-linux-musl-native/bin/musl-ranlib
+          echo "/x86_64-linux-musl-native/bin" >> $GITHUB_PATH
+          echo "RUSTFLAGS=-C link-arg=-static" >> $GITHUB_ENV
+          echo "CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_LINKER=/x86_64-linux-musl-native/bin/x86_64-linux-musl-gcc" >> $GITHUB_ENV
+
+      - name: Build Iroha
+        run: |
+          cargo build --target x86_64-unknown-linux-musl --features vendored --profile deploy
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
```

Comment:
This should not be necessary! The image should already come with a functional python interpreter.